### ISSUE TITLE
pip install mamonsu error

### ISSUE THREAD

**Issue by masikmos** [postgrespro/mamonsu#96]
hi guys.

i got error on all servers after pip install mamonsu

```
Collecting mamonsu
  Downloading mamonsu-2.3.0.tar.gz (85kB)
    100% |████████████████████████████████| 92kB 5.0MB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-uLtdtl/mamonsu/setup.py", line 17, in <module>
        long_description=long_description(),
      File "/tmp/pip-build-uLtdtl/mamonsu/setup.py", line 9, in long_description
        with codecs.open(path.join(filespath, 'README.rst'), encoding='utf8') as f:
      File "/usr/lib/python2.7/codecs.py", line 896, in open
        file = __builtin__.open(filename, mode, buffering)
    IOError: [Errno 2] No such file or directory: '/tmp/pip-build-uLtdtl/mamonsu/README.rst'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-uLtdtl/mamonsu/
```

Help! )

**Comment by horkko** [comment]
👍 got the same issue some weeks ago. To get it working, I've cloned the repo

**Comment by gsmol** [comment]
fixed in 2.3.1, please try to upgrade your installation
